test/e2e/auth/service_accounts.go

```diff
@@ -153,6 +153,11 @@ var _ = SIGDescribe("ServiceAccounts", func() {
 		}
 	})
 
+	/*
+		Release : v1.9
+		Testname: Service Account Tokens Must AutoMount
+		Description: Ensure that Service Account keys are  mounted into the Pod. Pod contains three containers each will read Service Account Key, root CA Key and Namespace Key respectively from the default API Token Mount path. All these three files MUST exist and the Service Account mount path MUST be auto mounted to the Pod.
```

Comment:
Please wrap this line around 80-100 characters

Reply:
I think gofmt was messing up with some of the indentation, it seem to be smart with comment body too. Please see if I addressed all your concerns.